extensions/interactions/ImageClickInput/directives/ImageClickInput.js

```diff
@@ -69,20 +96,20 @@ oppia.directive('oppiaInteractiveImageClickInput', [
               return 'inline';
             }
           };
+          $scope.$on(EVENT_NEW_CARD_AVAILABLE, function(evt, data) {
+            if (data) {
+              $scope.interactionIsActive = false;
+            }
+          });
           $scope.onMousemoveImage = function(event) {
-            var image = $($element).find('.oppia-image-click-img');
-            $scope.mouseX = (event.pageX - image.offset().left) / image.width();
-            $scope.mouseY = (event.pageY - image.offset().top) / image.height();
-            $scope.currentlyHoveredRegions = [];
-            for (var i = 0; i < imageAndRegions.labeledRegions.length; i++) {
-              var labeledRegion = imageAndRegions.labeledRegions[i];
-              var regionArea = labeledRegion.region.area;
-              if (regionArea[0][0] <= $scope.mouseX &&
-                  $scope.mouseX <= regionArea[1][0] &&
-                  regionArea[0][1] <= $scope.mouseY &&
-                  $scope.mouseY <= regionArea[1][1]) {
-                $scope.currentlyHoveredRegions.push(labeledRegion.label);
-              }
+            if ($scope.interactionIsActive) {
```

Comment:
Return early if not active -- see above.

Reply:
Done